README.md

```diff
@@ -311,7 +311,22 @@ mv ~/.cache/yay/* ~/.local/share/pikaur/aur_repos/
 find ~/.local/share/pikaur/aur_repos -mindepth 1 -maxdepth 1 -type d | xargs -r -I '{}' -- sh -c 'cd "{}" && git rev-parse HEAD > last_installed.txt'
 ```
 
+##### How to downgrade a package?
 
+This will show a list of commits to choose one to downgarade to.
+
+```sh
+pikaur -G <package> # choose commit from the list
```

Comment:
from which list?

Reply:
I've updated the documentation, can you please re-check :)

Thanks for your patience in advance :)